app/models/issue.rb

```diff
@@ -20,7 +20,7 @@ def get_interfaces(interface = nil)
 
   def get_headers(header = nil)
     all_headers = http_data(:headers)
-    return all_headers if header.nil?
+    return all_headers if header.nil? || all_headers == "<no information>"
     all_headers.find { |h| h[header] }.try(:values).try(:first)
```

Comment:
ahhhh... this is why i have returning strings for no values. I know we agreed we should keep this but really we should change it. Please stop returning strings for these. `all_headers` should be nil in this case. If we want to show some kind of string and use that method in mailers we should make a new one that checks if the returned data is nil.